main.go

```diff
@@ -16,10 +16,11 @@ import (
 var Options struct {
 	AssistedServiceScheme string `envconfig:"ASSISTED_SERVICE_SCHEME"`
 	AssistedServiceHost   string `envconfig:"ASSISTED_SERVICE_HOST"`
-	Port                  string `envconfig:"PORT" default:"8080"`
+	DataDir               string `envconfig:"DATA_DIR"`
```

Comment:
What is that?

Reply:
This was just a reordering.

The data directory is where we'll store the template images locally and the directory we use for scratch space while we're creating the minimal iso template.

Reply:
Yeah wanted to delete the comment after realizing it was juts reorder. Thanks for the explanation